apps/qubit/modules/term/actions/navigateRelatedComponent.class.php

```diff
@@ -54,6 +54,7 @@ static function getEsDocsRelatedToTerm($relatedModelClass, $term, $options = [])
 
     // Allow for options to override default behavior
     $search = !empty($options['search']) ? $options['search'] : new arElasticSearchPluginQuery();
+    $search->query->setSize(10000);
```

Comment:
Is this really needed only to get the count?

Reply:
Removed!